workflow-templates/im-deploy-az-app-manually.yml

```diff
@@ -141,7 +192,7 @@ jobs:
   attestor-approval:
     needs: [set-vars]
     runs-on: ubuntu-latest # Force this to run on github-hosted runner by using a tag that does not exist on self-hosted runners
-    environment: '${{ github.event.inputs.environment }} Attestor Approval' # Use inputs context because env context is not available to environment:
+    environment: '${{ github.event.inputs.environment-or-target }} Attestor Approval' # Use inputs context because env context is not available to environment:
```

Comment:
I think this should match the environment the workflow is using above.  For the approval groups they should be using the input or the github secrets env, not a mix.

Reply:
Good catch. Only saw stakeholder in other tf workflows.